test/AxelarGatewayMultisig.js

```diff
@@ -893,5 +894,117 @@ describe('AxelarGatewayMultisig', () => {
           });
       });
     });
+
+    describe('send token from gateway', () => {
+      it('should burn internal token and emit an event', async () => {
```

Comment:
We should test both types of internal token.

Reply:
We will make it a part of integration tests in the future